cmd/spiffe-helper/config/config_test.go

```diff
@@ -10,7 +11,7 @@ import (
 )
 
 func TestParseConfig(t *testing.T) {
-	c, err := ParseConfig("../../test/fixture/config/helper.conf")
+	c, err := ParseConfig("../../../test/fixture/config/helper.conf")
```

Comment:
mmm looks like this config is only used for this test, may we move all test files to a folder inside this package?

Reply:
Done